### ISSUE TITLE
Closing sequence not always being added to end of string

### ISSUE THREAD

**Issue by psmarsh** [jorgebucaran/clorox#14]
Hi! I'm not really sure how ANSI codes are supposed to work, so maybe this is intentional. But I noticed that 

```
console.log(`${
  c.bgGreen.red("What's").blue("up?")
}`)
```

and

```
console.log(`${
  c.red.bgGreen("What's").blue("up?")
}`)
```

produce different output. This is because in the first example, the bgGreen closing sequence never gets added to the end of the string, while in the second example it is.

The red closing sequence is never added as well in the second example, but that doesn't matter because the blue overrides it.

I believe this could be fixed by changing line 10 in index.js to 

`get: () => Clorox(old + ansi[name][0], (close || "") + ansi[name][1])`

I think that should work!

**Comment by jorgebucaran** [comment]
@psmarsh Could you send me a PR? :)

**Comment by psmarsh** [comment]
Done, it's my first one so I hope I did it right lol

**Comment by jorgebucaran** [comment]
Thank you, @psmarsh! Published as clorox@**1.0.3**. 🎉
